policy/recipe2policy.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+const STRING = 'stringPayload';
+
+class Attribute {
```

Comment:
TS might be nicer for implementing this tool/infra at some point, I feel like it'd save on a lot of JS boilerplate and so forth, but I get that this is a simple tool for now, just commenting while reading.

Or if we have a specific reason to prefer JS here definitely let me know.